server/routes/unauthenticatedRoutes.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express'
+
+import paths from '../constants/paths'
+
+export default function routes(): Router {
+  const router = Router()
+
+  // Accessibility statement
+  router.get(paths.ACCESSIBILITY, (req, res) => {
+    res.render('pages/accessibilityStatement')
```

Comment:
This is where you wonder if the path should map to the rendered page template as well, and maybe a type then you could just loop round them passing in path and page. But that's a possible for another day 😁

Reply:
Yeah, that could be good - will resist the temptation to refactor it all now though unless/until we get a lot of these static pages!